dali/operators/imgcodec/host_decoder.cc

```diff
@@ -32,10 +32,26 @@ using HostDecoderCrop = WithCropAttr<HostDecoder, CPUBackend>;
 using HostDecoderSlice = WithSliceAttr<HostDecoder, CPUBackend>;
 using HostDecoderRandomCrop = WithRandomCropAttr<HostDecoder, CPUBackend>;
 
+DALI_REGISTER_OPERATOR(decoders__Image, HostDecoder, CPU);
+DALI_REGISTER_OPERATOR(decoders__ImageCrop, HostDecoderCrop, CPU);
+DALI_REGISTER_OPERATOR(decoders__ImageSlice, HostDecoderSlice, CPU);
+DALI_REGISTER_OPERATOR(decoders__ImageRandomCrop, HostDecoderRandomCrop, CPU);
+
+// Deprecated aliases: fn.image*_decoder
+DALI_REGISTER_OPERATOR(ImageDecoder, HostDecoder, CPU);
+DALI_REGISTER_OPERATOR(ImageDecoderCrop, HostDecoderCrop, CPU);
+DALI_REGISTER_OPERATOR(ImageDecoderSlice, HostDecoderSlice, CPU);
+DALI_REGISTER_OPERATOR(ImageDecoderRandomCrop, HostDecoderRandomCrop, CPU);
+
+// Deprecated aliases: fn.image*_decoder
 DALI_REGISTER_OPERATOR(experimental__decoders__Image, HostDecoder, CPU);
+DALI_REGISTER_OPERATOR(experimental__ImageDecoder, HostDecoder, CPU);
```

Comment:
Why add those now if we have not exposed them in this form before? We don't break any existing code by not having them.

![image](https://github.com/NVIDIA/DALI/assets/11878086/130f00a4-15c3-4fe7-9d2d-18fb1f885e97)